edx_exams/apps/core/models.py

```diff
@@ -44,9 +45,6 @@ class Meta:
     def get_full_name(self):
         return self.full_name or super().get_full_name()
 
-    def __str__(self):
```

Comment:
when printing user we're usually going to prefer the default behavior (username) for admin pages, logging, etc so I've removed this.